pkg/sql/opt/memo/testdata/stats/scan

```diff
@@ -829,16 +829,16 @@ SELECT * FROM hist WHERE (d >= 5 AND d < 12) OR d >= 40
 ----
 index-join hist
  ├── columns: a:1(int) b:2(date) c:3(decimal) d:4(float!null) e:5(timestamp) f:6(timestamptz) g:7(string)
- ├── stats: [rows=131, distinct(4)=8.5, null(4)=0]
- │   histogram(4)=  0          0          45   10   36          0           0   40
+ ├── stats: [rows=120, distinct(4)=8.11111, null(4)=0]
```

Comment:
This change looks suspicious, but it appears to be _more_ correct now than it was.

Reply:
Hmm, I might be wrong. This is a float type so the number of values in `(0, 10)` is closer to 10 than it is to 9, like it would be if it was an integer...